rpc/common/moc_common_common.proto

```diff
@@ -350,6 +350,7 @@ enum HyperVGeneration {
 enum DiskFileFormat {
 	DiskFileFormatVHDX =  0;
 	DiskFileFormatVHD = 1;
+	DiskFileFormatUNKNOWN = 2;
```

Comment:
Are we expecting cx to actually input Unkonwn. I couldn't really think of situation where this is needed. especially when default value for int32 is 0 (VHDX).

Reply:
if we dont have cx flow that would allow this value to be passed down, This should be removed.